Src/.env

```diff
@@ -0,0 +1,10 @@
+FROM_EMAIL=noreply@deurgenta.ro
+SENDGRID_API_KEY=
+ASPNETCORE_URLS=http://+:80
+DB_CONNECTION_STRING=Server=postgres;Port=5432;Database=de-urgenta-db;User Id=docker;Password=docker;
+TEMPLATE_FOLDER=EmailTemplates
+SWAGGER_REDIRECT_URIS=https://localhost:5040/swagger/oauth2-redirect.html
+SWAGGER_CORS=https://localhost:5040
+IDENTITYSERVER_URL=http://localhost:5042
+HANGFIRE_USERNAME=admin
+HANGFIRE_PASSWORD=demo1234
```

Comment:
not really sure we want to keep this file in repo.. 

@catileptic , @tudoramariei , feedback pls

Reply:
sorry ,yes I will add to docker ignore